xwiki-platform-core/xwiki-platform-repository/xwiki-platform-repository-test/xwiki-platform-repository-test-utils/src/main/java/org/xwiki/repository/test/RepositoryTestUtils.java

```diff
@@ -21,12 +21,9 @@
 
 import java.io.File;
 import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
static imports are fine. It's just that the rule is to not import more than needed.